src/app/course/course-registration/course-register.component.ts

```diff
@@ -64,6 +66,10 @@ export class CourseRegisterComponent implements OnInit {
     getCourseName(): void {
         this.courseService.getCourse(this.courseId).subscribe(course => {
             this.courseName = course.name;
+            // this.showSkeletons = false;
```

Comment:
Can we please remove the `showSkeletons` if it's not being used? Same for the commented code.